src/molecules/content/ComparatorSingleModal.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { StyleSheet, View, Text } from 'react-native';
+import { registeredTextStyle, UrbiFontStyles } from 'src/utils/textStyles';
+import { layoutStyles } from 'src/utils/styles';
+
+const styles = StyleSheet.create({
+  Wrapper: {
+    height: 72,
+    ...layoutStyles.ColumnJustifyStart,
+    alignItems: 'flex-start',
+    width: '60%',
+  },
+  Title: registeredTextStyle('title' as keyof UrbiFontStyles),
+  Content: {
+    ...layoutStyles.RowAlignCenter,
+    justifyContent: 'flex-start',
+  },
+  BottomLabel: {
+    ...registeredTextStyle('body' as keyof UrbiFontStyles),
+    marginTop: 4,
+  },
+});
+
+export type ComparatorSingleModalProps = {
+  content: JSX.Element;
```

Comment:
throughout the DS, we try to limit this to very few cases. Do we have a shortlist of components that can go in here? If we do, we usually prefer to have them, so that the lib user knows what kind of content is expected to be set inside this component just by looking at its props (and also, TS helps us prevent unexpected components here)

Reply:
@micheleb Thanks. This is a big change as I moved the [PathToDestination](https://github.com/urbi-mobility/urbi-react-native/blob/19edb04586a452a10778b9d3a20b43608c60d424/Urbi/layouts/comparator/PathToDestination.tsx) logic in the UI Library. 

Related Commits b1b906a c3cd127
https://github.com/urbi-mobility/urbi-react-native/pull/233/commits/92b8042df431429dc20847b19eae1dfaf785e21e https://github.com/urbi-mobility/urbi-react-native/pull/233/commits/cacd398405944e92bbf927f8b169d29bb5cafd9b